### DISCUSSION TITLE
I have an error while merging.

### DISCUSSION THREAD

**Discussion by MkDev11** [community/community#67414] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In the past, i created new repo

Then i commit files to master branch and pushed it to new repo

Today i created new branch 'develop' and commit files which was not just ones in the past to that branch.

As a result, i couldn't merge two files.

It caused errors which said that two files were not similar at all.

Please help me with it.

**Comment by ZhengLinLei** [comment]
Do this steps:
* `git checkout master`
* `git pull origin master`
* `git checkout develop`
* `git merge master`

Doing this you will get back an issue about a problem ocurred during merging, only have to fix it. Then push the branch `depelop` to server again.
